packages/v4/src/content/design-guidelines/usage-and-behavior/pagination/pagination.md

```diff
@@ -4,42 +4,48 @@ section: usage and behavior
 hideTOC: true
 showTitle: true
 ---
-Pagination is used when you want to break up data into multiple pages. It is typically used within a table, list or card view.  If you choose to use infinite scrolling, then pagination would not be necessary. 
+Pagination splits data across multiple pages, typically within a table, list, or card view. If you use infinite scrolling, you don't need to use pagination.
+
+Add pagination above and below content views so users can easily navigate as they read through listed items on a page.
 
 ![top-bottom-pagination](./img/top-bottom-pagination.png)
 
-**1. Top pagination:** Pagination is always right-aligned in the toolbar. If pagination is required, we recommend using pagination both above and below content views.
+**1. Top pagination:** Always right-align pagination in the top toolbar, above content views.
 
-**2. Bottom pagination:** Pagination should be shown below the content view as well.
+**2. Bottom pagination:** Right-align pagination below content views, too.
 
 ## Full pagination
 
-If you do not have to worry about spatial limitations within the top toolbar, we recommend you use full pagination. By adding this to your toolbar, you will have to limit the other actions added in the toolbar on desktop and tablet views. The full variation does condense into only an item count upon mobile.
+Use full pagination unless you're restricted to a narrow workspace. Full pagination occupies more toolbar territory, so you'll have limited room for other items like bulk selectors, buttons, filters, or input fields. 
```

Comment:
I would add "then use **compact pagination.** after "Use full pagination unless you're restricted to a narrow workspace, " and then link to the compact pagination. 

You would do it like this in markdown [compact pagination](#compact-pagination)

Reply:
Great, adding this in.